src/ag_main.ml

```diff
@@ -78,7 +78,11 @@ let main () =
     let l = Str.split (Str.regexp " *, *\\| +") s in
     opens := List.rev_append l !opens
   in
+  let type_convs = ref [] in
   let options = [
+    "-type-conv", Arg.String (fun s ->
+      type_convs := Str.split (Str.regexp ",") s
+    ), "Type conv stuff";
```

Comment:
A better message would be good, so that:
* type-conv users know the syntax
* people who don't know about type-conv don't have to worry about what this is

Something like: 
```ocaml
"GEN1,GEN2,...
          Insert 'with GEN1, GEN2, ...' after OCaml type defintions for the type-conv preprocessor"
```